gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -181,15 +195,27 @@ def create_pangolin_grch38_ht() -> hl.Table:
 
     There's no precomputed score for all possible variants, the scores were
     generated for gnomAD v4 genomes (=v3 genomes) and v4 exomes variants in
-    gene body only.
+    gene body only with improved code of developers at Invitae:
```

Comment:
```suggestion
    gene body only with code from developers at Invitae:
```

Reply:
I think unless we say why its improved we can drop that adjective...this is just my opinion